### PR TITLE
update web remocon pairing mode

### PR DESCRIPTION
**Pull request by dwlee**
update for supporting web remocon pairing mode.
It need to following pull-requests for test.

https://github.com/RobotWebTools/rosbridge_suite/pull/172
https://github.com/RobotWebTools/roslibjs/pull/159

### CODE REVIEW

File: rocon_web_remocon/www/index.js [hunk 1]
```diff
@@ -8,11 +8,29 @@ var ros = new ROSLIB.Ros();
 var gListRoles = [];
 var gListInteractions = [];
 var gFinalUrl;
+var gFinalHash;
 
 var gUrl;
 var gCookieCount;
 
-var defaultUrl;
+var defaultUrl = 'ws://192.168.10.24:9090';
```
Comment (jihoonl) [inline rocon_web_remocon/www/index.js]: Please keep default URL empty.

File: rocon_web_remocon/www/index.js [hunk 2]
```diff
@@ -8,11 +8,29 @@ var ros = new ROSLIB.Ros();
...
+
+
+//Remocon profile
+var gPublishers = {}
+var gRunningInteractions = [];
+var gRoconVersion = 'acdc'; //todo make rocon/version.js fot obtaining
+var gRemoconUUID = generateUUID().replace(/-/g,'');
+var gRemoconName = 'web_remocon_' + gRemoconUUID;
+var gRemoconRoconURI = 'rocon:/*/' + gRemoconName + '/*/trusty|chrome'
```
Comment (jihoonl) [inline rocon_web_remocon/www/index.js]: should utilise [getBrowser](https://github.com/robotics-in-concert/rocon_web_gui/blob/indigo/rocon_web_remocon/www/index.js#L577-L586) to get correct rocon uri.

File: rocon_web_remocon/www/index.js (hunk 1 above)
Comment (dwlee) [inline rocon_web_remocon/www/index.js]: I updated this comment

File: rocon_web_remocon/www/index.js (hunk 2 above)
Comment (dwlee) [inline rocon_web_remocon/www/index.js]: I also updated this comment

**Comment by dwlee** [comment]
closes #11

**Comment by dwlee** [comment]
Also closed #6

**Pull request merged**